doc/command_ref.rst

```diff
@@ -1117,7 +1119,8 @@ packages to those matching the specification. All packages are considered if no
     Limit the resulting set only to packages that conflict ``<capability>``.
 
 ``--whatenhances <capability>``
-    Limit the resulting set only to packages that enhance ``<capability>``.
+    Limit the resulting set only to packages that enhance ``<capability>``. If you want to limit to
+    all dependencies, use the \-\ :ref:`-whatdepends <whatdepends_optin-label>` option.
```

Comment:
I think the added sentence is strictly speaking gramatically wrong and also doesn't provide enough information about what adding the switch should actually achieve.

I did try the commands from the referenced bug, but didn't get anything listed on f29 no matter what I did, might be reproducible on f30 only?

Perhaps if I was more familiar with the semantics of the options I would understand it, but I don't. That probably makes me a good reviewer for this :smile: Anyway, not understanding the semantics, I can't really suggest better wording, but I think the added sentence above needs to be more specific about what it does and potentially it might be a good idea to extend the --whatdepends description with basically the same information, kind of coming from the other direction.

Reply:
What about "Refer to `--whatdepends` if you want to list all depending packages"

Reply:
Oh.. so I somehow thought `--whatdepends` was somehow supposed to be used along with the various options in question and didn't understand how. Sorry.

So I'd just go with "Use `--whatdepends` if you want to list all depending packages", yeah. Although it seems a bit redundant to me, but the bug report is obviously about this not being there. Also, still the bug was about weak dependencies originally, and as @j-mracek said there they are handled differently by the various options, so this should be described too?